guide/daily-spending-wallet/send-fees.md

```diff
@@ -0,0 +1,205 @@
+
+</div>
+
+## On-chain fees
+
+<div class="center" markdown="1">
+
+{% include picture.html
+   image = "/assets/images/guide/daily-spending-wallet/sending/send-fees/onchain-fee-sample.png"
+   retina = "/assets/images/guide/daily-spending-wallet/sending/send-fees/onchain-fee-sample@2x.png"
+   alt-text = "How to display an onchain transaction fee"
+   width = 400
+   height = 252
+   layout = "float-right-desktop"
+%}
+
+This fee is largely dependent on how many other transactions are currently [waiting]({{ '/guide/glossary/#mempool' | relative_url }}) to be processed on the base layer as a whole. The [average fee](https://ycharts.com/indicators/bitcoin_average_transaction_fee) in January 2021 was $0.63, and $28.60 in April 2021.
+
+A new block of transactions is confirmed roughly every 10 minutes. Blocks have size limits, which places a limit on the number of transactions they can contain. Transactions size itself varies with complexity. A complex multi-key transaction with many inputs and outputs involves more data, and therefore carries a higher fee, than a simple one that involves only one input and output. An organic fee market emerges from these dynamics, as miners try to earn as much as they can from fees, and transacting users try to optimize for time, fee rate, or other priorities.
+
+</div>
+
+<div class="center" markdown="1">
+
+{% include picture.html
+   image = "/assets/images/guide/daily-spending-wallet/sending/send-fees/fee-options.png"
+   retina = "/assets/images/guide/daily-spending-wallet/sending/send-fees/fee-options@2x.png"
+   modalImage = "/assets/images/guide/daily-spending-wallet/sending/send-fees/fee-options-big.png"
+   alt-text = "Screen showing fee options for an onchain transaction"
+   caption = "A simple framework for on-chain fees is the urgency of the transaction."
+   width = 250
+   height = 541
+   layout = "float-right-desktop -background -shadow"
+%}
+
+Generally, it is best to dynamically calculate a reasonable fee range and provide a few simple options based on how urgent the transaction is for the user. Total cost and confirmation time are often the primary decision points.
+
+</div>
+
+#### High fee situations
+
+<div class="center" markdown="1">
+
+{% include image.html
+   image = "/assets/images/guide/daily-spending-wallet/sending/send-fees/fee-options-high.png"
+   retina = "/assets/images/guide/daily-spending-wallet/sending/send-fees/fee-options-high@2x.png"
+   alt-text = "Screen showing a fee picker for an onchain transaction with high fee options marked in red"
+   width = 250
+   height = 541
+   layout = "float-right-desktop -background -shadow"
+%}
```

Comment:
I noticed that you included `fee-options-high-big.png` in the new image files, but did not make this make this one a clickable modal.

Reply:
Good catch. Fixed.